pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -148,6 +148,16 @@ func isRequiredConfigPresent(config []byte) error {
 		if configValString, ok := configVal.(string); ok && len(configValString) == 0 {
 			return fmt.Errorf("%v empty in config", strings.Join(requiredPath, "."))
 		}
+
+		if requiredPath[0] == "etcd-servers" {
```

Comment:
I'd be curious to see in the tests if this also gated rev 1, though I doubt it will be the case. Worst case scenario we can always add that check as a precondition to the revision controller.

Reply:
Right, `requiredPath[0]` is `apiServerArguments`, updated the code and added unit tests

Reply:
My comment was about checking in the test run if the config for revision 1 got the required list of etcd-servers or if it bypassed this check. We need to make sure that rev 1 gets the minimum number of endpoints, otherwise we might run into the same problem as before with the first kube-apiserver pod.

Reply:
Ah, got it. Well, we can't issue a revision if config is not considered correct, so I think we have this covered.

Not sure we already have revision controller unit tests